### PR TITLE
set_dot implementation

### PR DESCRIPTION
**Pull request by guilhermebodin**
This way we can query the dual coefficients for the matrix A^T using set_dot, and it is easier to extend to new cones

### CODE REVIEW

**Comment by guilhermebodin** [comment]
This closes #26

File: src/dual_model_variables.jl [hunk 1]
```diff
@@ -31,6 +31,7 @@ function _add_dual_variable(dual_model::MOI.ModelLike, primal_model::MOI.ModelLi
                                                     S <: MOI.AbstractSet}
 
     row_dimension = get_ci_row_dimension(primal_model, ci) 
+    #Change to add_constrained_varibales in MOI 0.9.0
```
Comment (blegat) [inline src/dual_model_variables.jl]: Add a link to the issue

File: src/dual_equality_constraints.jl [hunk 2]
```diff
@@ -110,66 +111,22 @@ end
...
-                    push_to_scalar_affine_terms!(scalar_affine_terms, 2*one(T), dual_vi)
-                end
-            end
-        end
-    end
-    return 
+function set_dot(i::Int, s::MOI.AbstractVectorSet, T::DataType)
+    vec = zeros(T, MOI.dimension(s))
+    vec[i] = one(T)
+    return MOIU.set_dot(vec, vec, s)
```
Comment (blegat) [inline src/dual_equality_constraints.jl]: We should modify `MOIU.set_dot` to accept `AbstractArray` so that `vec` can be passed as a sparse vector

Comment (guilhermebodin) [reply src/dual_equality_constraints.jl]: I don't know if this is super simple, I can try it tomorrow

Comment (blegat) [reply src/dual_equality_constraints.jl]: Thanks for the PR! Add a comment to change this once https://github.com/JuliaOpt/MathOptInterface.jl/pull/805 is merged and released.

**Pull request merged**